### PR TITLE
[Chore] [WIP] Fix and reformat javadoc @author tag

### PR DESCRIPTION
**Pull request by axmmisaka**
See #1477 for more context, the questions are:
1. Do we want to keep e-mail or only keep name, effectively eliminating the problem?
2. Do we use `{@literal }` or `&lt; &gt;`?
3. Why are emails not showing in the generated Javadoc
4. Why is javadoc build failing

### CODE REVIEW

**Review by cmnrd — COMMENTED** [review]
I think using `{@literal }` is a great solution. How does this look in the generated rendered html?

**Comment by lhstrh** [comment]
The description of the PR seems to suggest that JavaDoc generation is broken?

**Comment by axmmisaka** [comment]
Yeah, doesn't appear to be working as I can't find `@author` tags in generated HTML plus `./gradlew javadoc` is spitting out tons of errors on java files in `src-gen`

**Comment by axmmisaka** [comment]
`./gradlew javadoc` doesn't produce javadoc with author tag (and generate docs for files under `src-gen` which doesn't appear to be right), and my intellij generate javadoc is not working, F

File: org.lflang.tests/src/org/lflang/tests/compiler/LinguaFrancaValidationTest.java [hunk 1]
```diff
@@ -63,11 +63,11 @@
...
  * 
- * @author{Edward A. Lee <eal@berkeley.edu>}
- * @author{Marten Lohstroh <marten@berkeley.edu>}
- * @author{Matt Weber <matt.weber@berkeley.edu>}
- * @author(Christian Menard <christian.menard@tu-dresden.de>}
- * @author{Alexander Schulz-Rosengarten <als@informatik.uni-kiel.de>}
+ * @author Edward A. Lee
+ * @author Marten Lohstroh
+ * @author Matt Weber
+ * @author (Christian Menard}
```
Comment (cmnrd) [inline org.lflang.tests/src/org/lflang/tests/compiler/LinguaFrancaValidationTest.java:69]:
```suggestion
 * @author Christian Menard
```

File: org.lflang/src/org/lflang/AttributeUtils.java [hunk 2]
```diff
@@ -47,9 +47,9 @@
 /**
  * A helper class for processing attributes in the AST.
  * 
- * @author{Shaokai Lin <shaokai@berkeley.edu>}
- * @author{Clément Fournier, TU Dresden, INSA Rennes}
- * @author{Alexander Schulz-Rosengarten <als@informatik.uni-kiel.de>}
+ * @author Shaokai Lin
+ * @author Clément Fournier, TU Dresden, INSA Rennes
```
Comment (cmnrd) [inline org.lflang/src/org/lflang/AttributeUtils.java:51]:
```suggestion
 * @author Clément Fournier
```

File: org.lflang/src/org/lflang/validation/AttributeSpec.java [hunk 3]
```diff
@@ -41,8 +41,8 @@
 /**
  * Specification of the structure of an attribute annotation.
  * 
- * @author{Clément Fournier, TU Dresden, INSA Rennes}
- * @author{Shaokai Lin <shaokai@berkeley.edu>}
+ * @author Clément Fournier, TU Dresden, INSA Rennes
```
Comment (cmnrd) [inline org.lflang/src/org/lflang/validation/AttributeSpec.java:44]:
```suggestion
 * @author Clément Fournier
```

**Review by cmnrd — APPROVED** [review]
This looks great! Thanks!

**Comment by cmnrd** [comment]
I think we should merge this. However, this also means we need to manually integrate the changes into https://github.com/lf-lang/lingua-franca/pull/1482... @a-sr What do you think?

**Comment by a-sr** [comment]
I agree. You can merge this. There are already some other conflicts in #1482 that need to be resolved manually. A few more will not change much.

**Pull request merged**